src/main/java/org/mariadb/jdbc/internal/com/read/OkPacket.java

```diff
@@ -70,6 +73,11 @@ public OkPacket(Buffer buffer) {
     insertId = buffer.getLengthEncodedNumeric();
     serverStatus = buffer.readShort();
     warnings = buffer.readShort();
+    if (buffer.remaining() > 0) {
+        message = buffer.readStringLengthEncoded(StandardCharsets.UTF_8);
```

Comment:
According to https://dev.mysql.com/doc/internals/en/packet-OK_Packet.html, if the session supports `CLIENT_SESSION_TRACK` then this is a `string<lenenc>` (length-prefixed string); otherwise, this is `string<EOF>` (string extending from here to the end of the packet).

This code shouldn't assume there's always a length prefix.